src/components/common/CardView/CardInfo/CardInfo.jsx

```diff
@@ -25,15 +25,15 @@ const CardInfo = (props) => {
       <div className="cardInfo-container__details">
         <div className="cardInfo-container__details-left">
           <div className="cardInfo-container-name">{`${firstName} ${lastName}`}</div>
+          {/* TODO Remove this hard coded professional details when get it from the api.*/}
+          <div className="cardInfo-container-profession">{profession || 'Looking for Opportunity' }</div>
```

Comment:
Since we are getting profession in the api response. Please remove this 'Looking for Opportunity' text and the TODO as well
```suggestion
          <div className="cardInfo-container-profession">{profession}</div>
```

Reply:
done